test/com/collections/MethodsTest.java

```diff
@@ -103,25 +103,64 @@ void reverseArrayList(){
         list2.add(employee2);
         list2.add(employee1);
 
-        assertEquals(methods.reverseArrayList(list1),list2);
+        assertEquals(methods.reverseArrayList(list1), list2);
     }
 
-    @Test void checkIfAnagram(){
-        assertTrue(methods.checkIfAnagram("dupa","upad"));
-        assertTrue(methods.checkIfAnagram("moszna","mszona"));
-        assertTrue(methods.checkIfAnagram("sieckarnia","kaseciarni"));
+    @Test
+    void checkIfAnagram() {
+        assertTrue(methods.checkIfAnagram("dupa", "upad"));
+        assertTrue(methods.checkIfAnagram("moszna", "mszona"));
+        assertTrue(methods.checkIfAnagram("sieckarnia", "kaseciarni"));
 
-        assertFalse(methods.checkIfAnagram("dupa","zupa"));
-        assertFalse(methods.checkIfAnagram("drzewo","JavaCollections"));
+        assertFalse(methods.checkIfAnagram("dupa", "zupa"));
+        assertFalse(methods.checkIfAnagram("drzewo", "JavaCollections"));
 
     }
 
-    @Test void canBePalindrome(){
+    @Test
+    void canBePalindrome() {
         assertTrue(methods.canBePalindrome("kajak"));
         assertTrue(methods.canBePalindrome("kkaaj"));
         assertTrue(methods.canBePalindrome("kkaa"));
         assertFalse(methods.canBePalindrome("abcd"));
         assertFalse(methods.canBePalindrome("kajakk"));
     }
 
+    @Test
+    void sumTwoIntArrays() {
+
+        //1 Case
+        int[] array1 = {1, 2, 3};
+        int[] array2 = {9, 9, 9};
+        int[] sumA1andA2 = {1, 1, 2, 2};
+        int[] methodSum1 = methods.sumTwoIntArrays(array1, array2);
+
+        if (sumA1andA2.length == methodSum1.length) {
```

Comment:
You can make some private method responsible for comparing two arrays. It can be reused then for each case.